backend/controller/observability/calls.go

```diff
@@ -0,0 +1,133 @@
+package observability
+
+import (
+	"context"
+	"github.com/TBD54566975/ftl/backend/controller/dal"
+	ftlv1 "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1"
+	"github.com/TBD54566975/ftl/backend/schema"
+	"github.com/TBD54566975/ftl/internal/log"
+	"github.com/TBD54566975/ftl/internal/model"
+	"github.com/alecthomas/types/optional"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+	"sync"
+	"time"
+)
+
+var callMeter = otel.Meter("ftl.call")
+var callInitOnce = sync.Once{}
+
+var callRequests metric.Int64Counter
+var callFailures metric.Int64Counter
+var callActive metric.Int64UpDownCounter
+var callLatency metric.Int64Histogram
+
+type CallBegin struct {
+	DestVerb *schema.Ref
+	Callers  []*schema.Ref
+}
+
+type CallEnd struct {
+	DeploymentKey model.DeploymentKey
+	RequestKey    model.RequestKey
+	StartTime     time.Time
+	DestVerb      *schema.Ref
+	Callers       []*schema.Ref
+	Request       *ftlv1.CallRequest
+	Response      optional.Option[*ftlv1.CallResponse]
+	CallError     optional.Option[error]
+}
+
+func initCallMetrics() {
+	callInitOnce.Do(func() {
```

Comment:
Why did you use a sync.Once here instead of just `var callRequests = ...` at the package scope?